Example/SygicMapsKit/Search examples/BrowseMapWithSearchResults.swift

```diff
@@ -60,7 +60,11 @@ class BrowseMapWithSearchResults: UIViewController, SYMKModulePresenter {
             browseMap.mapState.geoCenter = marker.coordinate!
             browseMap.mapState.zoom = 14
         } else if markers.count > 1 {
-            var boundingBox = SYGeoBoundingBox(bottomLeft: marker.coordinate!, topRight: markers[1].coordinate!)
+            guard let coord1 = marker.coordinate, let coord2 = markers[1].coordinate, coord1.isValid(), coord2.isValid() else { return }
```

Comment:
coord1 -> bottomLeftCoord
coord2 -> topRightCoord
?

Reply:
to nemusi byt tak uplne pravda